packages/bridge-ui-v2/src/libs/bridge/types.ts

```diff
@@ -171,6 +171,7 @@ export type RequireApprovalArgs = {
 export type ClaimArgs = {
   msgHash: Hash;
   message: Message;
+  receipt?: TransactionReceipt;
```

Comment:
If you add this here, you should omit it from the derived types such as releaseArgs.

Reply:
Updated. We don't need receipt.